### ISSUE TITLE
Build fails when VS 2013 is not installed

### ISSUE THREAD

**Issue by TeaDrivenDev** [fsprojects/Paket.VisualStudio#126]
### Description

The FAKE build fails when only Visual Studio 2015 is installed. The first hurdle is a missing `targets` file from VS 2013, which is easy to fix. The next one is the ILRepack step looking for 12.0 versions of VS assemblies, which I haven't found a solution for yet.

### Repro steps

1. Run `build.cmd`. Build fails with
```
The imported project "C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v12.0\VSSDK\Microsoft.VsSDK.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.
```

This can be fixed by adjusting the change made in #45 to `14`, which could either become the default at some point or be handled via a build parameter.

2. Run `build.cmd` again. Build fails with
```
Mono.Cecil.AssemblyResolutionException: Failed to resolve assembly: 'Microsoft.VisualStudio.CoreUtility, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
   at Mono.Cecil.BaseAssemblyResolver.Resolve(AssemblyNameReference name, ReaderParameters parameters)
   at Mono.Cecil.DefaultAssemblyResolver.Resolve(AssemblyNameReference name)
   at Mono.Cecil.MetadataResolver.Resolve(TypeReference type)
   at Mono.Cecil.TypeReference.Resolve()
   at ILRepacking.ReferenceFixator.FixReferences(Collection`1 attributes)
   at ILRepacking.ReferenceFixator.FixReferences(TypeDefinition type)
   at ILRepacking.Steps.ReferencesFixStep.Perform()
   at ILRepacking.ILRepack.Repack()
   at ILRepacking.Application.Main(String[] args)
Running build failed.
Error:
System.Exception: Error during ILRepack execution.
   at FSI_0005.Build.clo@134-12.Invoke(String message) in D:\Development\OpenSource\Paket.VisualStudio\build.fsx:line 134
   at FSI_0005.Build.clo@107-10.Invoke(Unit _arg6) in D:\Development\OpenSource\Paket.VisualStudio\build.fsx:line 134
   at Fake.TargetHelper.runSingleTarget(TargetTemplate`1 target) in C:\code\fake\src\app\FakeLib\TargetHelper.fs:line 492
```

I have not found any place where the version used there could be influenced. Changing all `12.0.0.0` target versions in `app.config` to `14.0.0.0` was unsuccessful.

### Expected behavior

The build is successful on a system that has only VS 2015 installed.

### Actual behavior

The project cannot be built on a system that has only VS 2015 installed.

### Related information 

* Operating system: Windows 10
* Branch: master
* .NET Runtime Version: 4.6.2

**Comment by forki** [comment]
please send PR ;-)

**Comment by TeaDrivenDev** [comment]
If I can figure out what Cecil's problem is. Step 0: Figure out what *Cecil* is.

**Comment by Reelix** [comment]
This is rather critical. VS2015 auto updated Paket in the background and has been causing our entire dev team issues the entire day. I've just stumbled across this post.  

Error	: Could not load file or assembly 'Mono.Cecil, Version=0.10.0.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756' or one of its dependencies. The system cannot find the file specified.	

100% to fail to do a Paket update with version 0.29.2

**Comment by forki** [comment]
what? this is about the build of the Paket.VisualStudio addin itself

how can that change anything in the behaviour? and this commit is not even released

**Comment by forki** [comment]
@Reelix so what your saying is: the addin in the current version is broken an reports Mono.Cecil issues?

**Comment by Reelix** [comment]
@forki This seems to be correct.

http://i.imgur.com/KPz4ZFG.png - On Update

New console app with nothing, the Paket VS extension, an empty paket.references file, and a paket.dependencies file with nothing except

source https://www.nuget.org/api/v2

It was initially blank, but it seems to have been auto-populated.

My apologies for hijacking this thread :)

**Comment by forki** [comment]
can you please test 0.29.3 - it seems to be an issue with binding redirects in the addin.

**Comment by Reelix** [comment]
@forki I can confirm that 0.29.3 updates successfully - Thanks for the amazingly fast response :D

**Comment by forki** [comment]
Cool. Thanks for reporting.

Am 22.11.2016 13:25 schrieb "Reelix" notifications@github.com:

**Comment by TeaDrivenDev** [comment]
Fixed by #127. I think this is only still open because my laptop broke a few hours after sending the PR.
